src/server/views/dashboard/macros.njk

```diff
@@ -149,15 +143,15 @@
 
             setTimeout(function () {
               if (isConfirming) {
-                alert("You have successfully confirmed " + title + "’s details meet the requirements to be added to the list.");
+                alert(`You have confirmed ${title} meets the requirements to be added to the list.`);
```

Comment:
What happens when js is disabled/fails to load?

Reply:
Nothing works